src/services/exchanges/base/composite.exchange.service.ts

```diff
@@ -21,12 +22,24 @@ export abstract class CompositeExchangeService extends FuturesExchangeService {
     try {
       const balances = await this.getBalances(account);
       const balance = balances.filter((b) => b.coin === symbol).pop();
+      if (balance == null) {
+        throw new TickerFetchError('balance missing');
```

Comment:
I'm not a big fan of putting strings outside constants, we should find a workaround for this.